packages/itwinui-react/src/core/Tree/TreeNode.test.tsx

```diff
@@ -220,3 +220,123 @@ it('should render selected node', () => {
   fireEvent.click(treeNode);
   expect(onSelected).toHaveBeenCalledWith('testId', false);
 });
+
+it('should render checkbox with  checkboxProps correctly', () => {
```

Comment:
nit: maybe some of these tests could be combined, since they're all testing similar things

Reply:
Combined the tests in this [commit](https://github.com/iTwin/iTwinUI/pull/2022/commits/b5bb153b230aadce93b20f3d2c2bae88a68671c7).

Reply:
less nitty: These tests are not properly testing the className merging currently. It would be better to test both the original `iui-` class and the custom class.

See example:

https://github.com/iTwin/iTwinUI/blob/442ff49f2502aabf5f9495262a6e4f1941804d3c/packages/itwinui-react/src/core/Label/Label.test.tsx#L50-L53

Reply:
Added testing for both classes in this [commit](https://github.com/iTwin/iTwinUI/pull/2022/commits/0889be47f673ed465d27f49a5327a8c084c78de8).